### ISSUE TITLE
Missing exposed mask keyword argument for BatchNormalization

### ISSUE THREAD

**Issue by rtatton** [keras-team/keras#14281]
https://github.com/keras-team/keras/blob/5a7a789ee9766b6a594bd4be8b9edb34e71d6500/keras/layers/normalization.py#L130

The constructor states `self.supports_masking = True`. However, the `mask` keyword argument is missing from the `call()` method signature. Since mean and variance are dependent on the batch size, the following (or something along the same line) need to be added as well:
```
msk = tf.expand_dims(mask, axis=-1, dtype='float32')
masked = tf.boolean_mask(inputs, msk)
# Calculate mean, variance, etc. as normal.
```
Considering there could be a significant amount of masking, I suspect it will help to use batch renormalization since it handles small batch sizes better than the standard normalization.

**Comment by sushreebarsa** [comment]
@rtatton Moving this issue to closed status as there has been no recent activity, in case you still face the error please create a new issue.Thanks!
